math/ginkgo/Portfile

```diff
@@ -50,26 +52,71 @@ configure.args-append \
                     -DGINKGO_DOC_GENERATE_PDF=OFF \
                     -DGINKGO_JACOBI_FULL_OPTIMIZATIONS=OFF \
                     -DGINKGO_WITH_CCACHE=OFF \
-                    -DGTEST_SRC_DIR=${prefix}/src/googletest/
+                    -DMPIEXEC_EXECUTABLE=${prefix}/bin/${mpi.exec}
 
-# Even though compilation succeeds, OMP module does not build correctly on PPC.
-if {${build_arch} ni [list ppc ppc64]} {
+if {${configure.ccache}} {
     configure.args-replace \
-                    -DGINKGO_BUILD_OMP=OFF -DGINKGO_BUILD_OMP=ON
+                    -DGINKGO_WITH_CCACHE=OFF \
+                    -DGINKGO_WITH_CCACHE=ON
 }
 
-if {[string match *clang* ${configure.compiler}]} {
-    configure.ldflags-append \
-                    -L${prefix}/lib/libomp \
-                    -lomp
+# Ginkgo does not support HWLOC on MacOS:
+if {${os.platform} ne "darwin"} {
+    depends_lib-append \
+                    port:hwloc
+    configure.args-append \
```

Comment:
Did you mean `configure.args-replace`?

Reply:
Yes, thank you for noticing. Will fix it now.

Reply:
Fixed.